rest/data/interface.go

```diff
@@ -44,6 +44,8 @@ type Connector interface {
 	// processing recalculation job has been scheduled for each type of
 	// test (project/variant/task/test combo).
 	ScheduleSignalProcessingRecalculateJobs(context.Context) error
+	// TriageChangePoints returns once all change points have been successfully marked
+	TriageChangePoints(ctx context.Context, changePoints map[string]string, status string) error
```

Comment:
what is this changepoint map?

Reply:
A map of perf result id to measurement being triaged. I tried this with a slice of anonymous structs, but was having some problems having the datatypes match up. The spirit of this is to:
* Pass the information around in a way that's agnostic of which layer you are in
* Not have to map the data over to new structs in each passthrough layer

How do you typically do that?

Reply:
*Update* Anonymous structs don't work since this is a separate package.

An option would be to define something in the `model` package; but defining a struct the REST API is expecting in `model` seems very wrong. And mapping over structs in each layer seems inefficient/a lot of boilerplate.

Reply:
I've moved the struct into `model` package; it still seems janky for the REST layer to know about db models.